site/src/patterns/message.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Message
+description: The message component in the Atlas Design System
+template: standard
+figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Fproto%2FamgDNGgGYy2v2Unc6Eet46%2FQ%2526A-Private-Messages%3Fpage-id%3D0%253A1%26type%3Ddesign%26node-id%3D337-533145%26viewport%3D-15969%252C-17033%252C0.75%26scaling%3Dmin-zoom%26starting-point-node-id%3D337%253A548136%26show-proto-sidebar%3D1%26mode%3Ddesign%26t%3DXzNRwbG95jzc2Fvd-1
+classType: Component
+classPrefixes:
+  - messages
```

Comment:
Let's remove these lines, since this is a pattern doc, not a component doc.

```suggestion
```

Reply:
removed